lib/Migration/Version030104Date20241011192830.php

```diff
@@ -28,7 +28,7 @@ public function __construct(
 	 */
 	public function postSchemaChange(IOutput $output, Closure $schemaClosure, array $options): void {
 		$value = $this->openAiSettingsService->getServiceUrl();
-		if ($value !== '') {
+		if ($value !== '' && !str_ends_with($value, '/v1')) {
```

Comment:
And what if someone has set `https://blabla/v1/`?

Reply:
ah I missed that `/` in a hurry :)